src/components/Autocomplete/Autocomplete.less

```diff
@@ -80,7 +80,7 @@
 		}
 
 		.lucid-DropMenu-Control {
-			z-index: 301;
+			z-index: @zindex-tooltip;
```

Comment:
Yeah, I toyed with that but ended up using this dashes are _supposed_ to denote groupings.